### ISSUE TITLE
Is there any papers associatived with your codes.

### ISSUE THREAD

**Issue by SeaHifly** [Arpit1371/matlab_non_uniformity_correction-OF_infrared_images#1]
I am wondering if there is a paper which presents your contributions and methods.

**Comment by notiska** [comment]
https://www.ipol.im/pub/art/2012/glmt-mire/article.pdf - a bit late, but this paper perhaps?

**Comment by Arpit1371** [comment]
Yes , sorry i could not catch your last comment . The code can be optimized further using multithreading
